src/lambda/vue/samInvokeComponent.vue

```diff
@@ -314,249 +570,91 @@ export const Component = Vue.extend({
             this.showAllFields = false
         },
     },
-    // `createElement` is inferred, but `render` needs return type
-    template: `<!--
-    * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-    * SPDX-License-Identifier: Apache-2.0
-    -->
+})
+</script>
 
-<!--This is an experimental template that is not used directly.  -->
+<style>
```

Comment:
Can we get rid of `unsafe-inline` if we move this to a separate file? Also worth considering so we don't have to copy CSS to different webviews.

Reply:
No since moving it to a separate file is just a build-time concern. I think I can get rid of it another way though.